stdlib/public/Platform/CMakeLists.txt

```diff
@@ -89,7 +94,7 @@ foreach(sdk ${SWIFT_SDKS})
 
     # If this SDK is a target for a non-native host, create a native modulemap
     # without a sysroot prefix. This is the one we'll install instead.
-    if(NOT "${SWIFT_SDK_${SWIFT_HOST_VARIANT_SDK}_PATH}" STREQUAL "/")
+    if(NOT "${SWIFT_SDK_${sdk}_PATH}" STREQUAL "/")
```

Comment:
@modocache, is this correct? Is it supposed to be the target SDK, not the host SDK?